exercises/space-age/space_age_test.rb

```diff
@@ -1,30 +1,24 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'space_age'
 
+# Test data version: 7c63e40
 class SpaceAgeTest < Minitest::Test
   DELTA = 0.01
 
-  def test_age_in_seconds
-    age = SpaceAge.new(1_000_000)
-    assert_in_delta 1_000_000, age.seconds, DELTA
-  end
```

Comment:
This method was missing from x-common, so I omitted it. If we need it, I can add it to x-common or manually hack it in before the ```test_age_on_earth``` method in space_age_cases.rb